optuna/distributions.py

```diff
@@ -108,6 +108,12 @@ class FloatDistribution(BaseDistribution):
     This object is instantiated by :func:`~optuna.trial.Trial.suggest_float`, and passed to
     :mod:`~optuna.samplers` in general.
 
+    .. note::
+        When ``step`` is not :obj:`None`, if the range :math:`[\\mathsf{low}, \\mathsf{high}]`
+        is not divisible by :math:`step`, :math:`\\mathsf{high}` will be replaced with
+        the maximum of :math:`k \\times \\mathsf{step} + \\mathsf{low} < \\mathsf{high}`,
```

Comment:
```suggestion
        is not divisible by :math:`\\mathsf{step}`, :math:`\\mathsf{high}` will be replaced
        with the maximum of :math:`k \\times \\mathsf{step} + \\mathsf{low} < \\mathsf{high}`,
```

nitpick

Reply:
Good catch, thank you! cb47e922b24e9217988ef7b84914d665b508a5ae.